scripts/jenkins/ardana/ansible/group_vars/all/crowbar.yml

```diff
@@ -1,3 +1,11 @@
 
+admin_scripts_path: "/root/scripts"
+
 admin_mkcloud_config_file: "/root/mkcloud.config"
 qa_crowbarsetup_log: "/root/qa_crowbarsetup.log"
+
+crowbar_fqdn: "{{ ardana_env }}.prv.suse.net"
```

Comment:
This is a bit misleading. `crowbar_fqdn` sounds like FQDN of crowbar/admin node.

Reply:
renamed to `cloud_fqdn`